spec/tests/honeybee_schedule_file.rb

```diff
@@ -0,0 +1,153 @@
+
+require_relative '../spec_helper'
+
+require 'fileutils'
+
+def set_simulation_parameters(openstudio_model, weather_name)
+
+  weather_file_path = openstudio_model.workflowJSON.findFile(weather_name).get.to_s
+
+  epw_file = OpenStudio::EpwFile.load(weather_file_path).get
+
+  # Set weather file data
+  weather_file = OpenStudio::Model::WeatherFile.setWeatherFile(openstudio_model, epw_file).get
+  weather_file.makeUrlRelative
+  weather_name = "#{weather_file.city}_#{weather_file.stateProvinceRegion}_#{weather_file.country}"
+
+  # Add or update site data
+  site = openstudio_model.getSite
+  site.setName(weather_name)
+  site.setLatitude(epw_file.latitude)
+  site.setLongitude(epw_file.longitude)
+  site.setTimeZone(epw_file.timeZone)
+  site.setElevation(epw_file.elevation)
+
+  # Remove all the Design Day objects that are in the file
+  openstudio_model.getObjectsByType('OS:SizingPeriod:DesignDay'.to_IddObjectType).each(&:remove)
+
+  # Load the ddy file
+  ddy_file = "#{File.join(File.dirname(weather_file_path), File.basename(weather_file_path, '.*'))}.ddy"
+  ddy_model = OpenStudio::EnergyPlus.loadAndTranslateIdf(ddy_file).get
+
+  # Add the ddy objects to the existing model
+  ddy_model.getDesignDays.sort.each {|d| openstudio_model.addObject(d.clone) }
+
+  # Set the run period
+  run_period = openstudio_model.getRunPeriod
+  run_period.setBeginMonth(1)
+  run_period.setBeginDayOfMonth(1)
+  run_period.setEndMonth(12)
+  run_period.setEndDayOfMonth(31)
+
+  # Set the simulation control
+  simulation_control = openstudio_model.getSimulationControl
+  simulation_control.setDoZoneSizingCalculation(false)
+  simulation_control.setDoSystemSizingCalculation(false)
+  simulation_control.setDoPlantSizingCalculation(false)
+  simulation_control.setRunSimulationforSizingPeriods(true)
+  simulation_control.setRunSimulationforWeatherFileRunPeriods(true)
+
+end
+
+
+RSpec.describe Honeybee do
+
+  it 'can translate model energy fixed interval to schedule file' do
+    epw_name = 'USA_CO_Golden-NREL.724666_TMY3.epw'
+    epw_dir = File.join(File.dirname(__FILE__), '..', 'samples', 'epw')
+    schedule_dir = File.join(File.dirname(__FILE__), '..', 'output', 'schedule_osms')
+    schedule_file_dir = File.join(schedule_dir, 'schedule_file')
+
+    # translate to schedule file
+    if File.exists?(schedule_file_dir)
+      FileUtils.rm_rf(schedule_file_dir)
+    end
+    FileUtils.mkdir_p(schedule_file_dir)
+
+    openstudio_model = OpenStudio::Model::Model.new
+    workflow = openstudio_model.workflowJSON
+    workflow.addFilePath(schedule_file_dir)
+    workflow.addFilePath(epw_dir)
+    workflow.setWeatherFile(epw_name)
+    workflow.saveAs(File.join(schedule_file_dir, 'in.osw'))
+    openstudio_model.setWorkflowJSON(workflow)
+
+    file = File.join(File.dirname(__FILE__), '../samples/model/model_energy_fixed_interval.json')
+    honeybee_obj_1 = Honeybee::Model.read_from_disk(file)
+    honeybee_obj_1.set_schedule_csv_dir(schedule_file_dir)
```

Comment:
@chriswmackey if `schedule_csv_dir` is set on the Honeybee model then we will translate to schedule file, we could add this parameter to the Honeybee JSON if you wanted?